File: calculaPreco.py
# Online Python compiler (interpreter) to run Python online.
# Write Python 3 code in this online editor and run it.
# Define função para calcuclar preço final
def calcular_preco_final(nome, preco, desconto=0):
    preco_final = preco - (preco * desconto / 100)
    print(f"Preço final de {nome}: R$ {preco_final:.2f}")

# Exemplo de uso
calcular_preco_final("Fone de ouvido", 200, 15)
print("Try programiz.pro")

File: calculaPreco.js
// Online Javascript Editor for free
// Write, Edit and Run your Javascript code using JS Online Compiler
function calcularPrecoFinal(produto) {
  const desconto = produto.desconto ?? 0;

  const precoFinal = produto.preco - (produto.preco * desconto / 100);

  console.log(`Preço final de ${produto.nome}: R$ ${precoFinal.toFixed(2)}`);

}

// Exemplo de uso

const produto1 = {

  nome: "Fone de ouvido",

  preco: 200,

  desconto: 15

};

calcularPrecoFinal(produto1);
console.log("Try programiz.pro");
